vars/runIntegTests.groovy

```diff
@@ -0,0 +1,27 @@
+void call(Map args = [:]) {
+    git url: 'https://github.com/opensearch-project/opensearch-build.git', branch: 'main'
+
+    def lib = library(identifier: 'jenkins@20211123', retriever: legacySCM(scm))
+    String manifest = args.manifest ?: "manifests/${INPUT_MANIFEST}"
+    def inputManifest = lib.jenkins.InputManifest.new(readYaml(file: manifest))
+
+    String tarball = args.architecture == "x64" ? env.ARTIFACT_URL_linux_x64 : env.ARTIFACT_URL_linux_arm64
```

Comment:
Right now the `architecture` parameter is used to determine the root url, which is used to determine the build manifest

Reply:
Hmm, I don't love it - because when we add more platforms + architectures we have to copypasta a bunch of code in a bunch of different places but we already have to deal with this problem with the 2 build legs.  Lets not try to solve it now.